### DISCUSSION TITLE
Request and response review for multiple targets

### DISCUSSION THREAD

**Discussion by sickwell** [projectdiscovery/nuclei#765] [Q&A] [answered]
When starting scanning with nuclei, is it possible to save each request and response to output log?
I know about debug feature, but this prints all in terminal.

**Comment by sickwell** [comment]
btw, this is imposible to save -debug to file. pipe not working too...

**Reply by ehsandeep** [reply]
Additionally you can redirect all the debug output to file without printing it with `-debug 2> nuclei_logs`

**Reply by sickwell** [reply]
thanks, forgot about this option!

**Comment by ehsandeep** [comment, marked as answer]
@sickwell you can use [proxify](https://github.com/projectdiscovery/proxify) for this, for example - 

![image](https://user-images.githubusercontent.com/8293321/121746747-f59e0580-cb23-11eb-9946-856db1f339f8.png)

**Reply by sickwell** [reply]
proxify failed after significant number of requests - tried with regular http and socks5 proxy.

**Reply by ehsandeep** [reply]
@sickwell what you exactly mean by failed? did you notice any error or it stopped working?
